spec/controllers/submit_source_controller_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+describe SubmitSourceController do
+  render_views
+
+  specify 'form works' do
+    get :new
+    response.should be_success
+  end
+
+  specify 'successful form post delivers mail' do
+    post :create, submit_source: {
+      url: 'http://www.example.com/url',
+      email: 'info@example.com',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.